packages/api/src/utils/car.js

```diff
@@ -1,5 +1,6 @@
 import { CID } from 'multiformats'
 import { CarWriter } from '@ipld/car'
+import { Blob } from '@web-std/blob'
```

Comment:
can we just use node 18 https://nodejs.org/dist/latest-v18.x/docs/api/globals.html#class-blob ?